src/selection.ts

```diff
@@ -7,15 +7,15 @@ import {ParameterName} from './parameter';
 import {Dict} from './util';
 
 export const SELECTION_ID = '_vgsid_';
-export type SelectionType = 'point' | 'interval';
+export type SelectionType = 'point' | 'interval' | 'lasso';
 export type SelectionResolution = 'global' | 'union' | 'intersect';
 
 export type SelectionInit = PrimitiveValue | DateTime;
 export type SelectionInitInterval = Vector2<boolean> | Vector2<number> | Vector2<string> | Vector2<DateTime>;
-
+export type SelectionInitLasso = number[];
 export type SelectionInitMapping = Dict<SelectionInit>;
 export type SelectionInitIntervalMapping = Dict<SelectionInitInterval>;
-
+export type SelectionInitLassoMapping = Dict<SelectionInitLasso>;
```

Comment:
Can these selections be initialized? I can imagine populating them with IDs somehow — though doing so will fall outside the scope of this PR, and is something we should think about generally for all ID-driven selections (e.g., `point` and geo-intervals). Putting that aside, I'm not sure it'd make sense to initialize the lasso specifically (i.e., pixel coordinates)? 

I'm guessing @dvmoritzschoefl, you were following a similar thought process because I don't see any additional references to these two types? So perhaps we can delete them, and make sure the lasso selection type doesn't allow `value`?

Reply:
True, I forgot to delete these 2.
I dont think there is a way to initialize the store anyway now, or am I missing something here?